frontend/mr-admin-flate/src/components/utbetaling/DelutbetalingRow.tsx

```diff
@@ -182,6 +204,16 @@ export function DelutbetalingRow({
       <Table.DataCell className={cellClass}>
         {formaterNOK(tilsagn.beregning.output.belop)}
       </Table.DataCell>
+      <Table.DataCell className={cellClass}>
+        <Checkbox
+          hideLabel
+          readOnly={!kanRedigere}
+          checked={frigjorTilsagn}
+          onChange={() => onFrigjorCheck(frigjorTilsagn)}
```

Comment:
Her burde vi kanskje heller sende inn ny verdi, i stedet for aktiv verdi, så slipper vi å negere verdien inne i `onFrigjorCheck`